CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+# iwant-bot
+---
+## Hello everyone!
+- :star2: If you are reading this document then I'm assuming that you are interested in contributing to the __iwant-bot__ project.
+- This project is made by Kiwi.com interns. 
+- All contributions are welcome: new use-cases, code, patches, bugfixes, issues, etc. 
+- This is just a set of guidelines, not rules. Use your best judgement, and feel free to propose changes in a pull request.
+
+
+## Ways to contribute
+---
+  - At first, check existing [issues] and our [pull requests] and try to find something 
+  - [Create an issue], if you think you have a good idea and let us consider your contribution.
+  - To report a bug:
+      - Open an issue with the label "bug" :bug:.

```

Comment:
i think external people can't label, also let's remove the trailing dots? i think it looks nicer without